src/components/ActionWrapper.js

```diff
@@ -59,14 +60,14 @@ class ActionWrapper extends Component {
             <FooterContainer {...this.props} />
           </div>
         </Router>
-      </FetchingContext.Provider>
+      {/*</FetchingContext.Provider>*/}
+      </div>
     );
   }
 }
 
 ActionWrapper.propTypes = {
   redirect: PropTypes.string,
-  is_fetching: PropTypes.bool
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it

Reply:
The actionWrapper does not need to know whether the app is fetching any more; just the buttons need to know